src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/CacheHealthDecider.java

```diff
@@ -121,8 +121,6 @@ public Decision operate() {
   }
 
   private void configureModifyCacheActionPriority() {
-    // Assigning shard request cache higher priority over field data cache
```

Comment:
nit: It would be nice to have an updated comment on line 124. What does this priority mean?